### ISSUE TITLE
Get 2 dimensional document vectors?

### ISSUE THREAD

**Issue by MrPaulAlbert** [ddangelov/Top2Vec#280]
Awesome project!  Any suggestions on how to report a document vector (ideally, reduced to 2 dimensions?).

**Comment by MrPaulAlbert** [comment]
Awesome!  

So, I had to change 'plt.plt(x,y)' to 'plt.plot(x,y)' but got this to work.  Unfortunately, I end up with a pretty messy hairball (193 topics).

Any idea how I can recreate this process using a model that I then reduce using "model.hierarchical_topic_reduction(10)"?

Thanks!

I get a pretty messy hairball, but I can work with that

**Comment by sean-doody** [comment]
> Thanks so much!
> 
> So if my reduced model has 10 topics, should I set the umap function to "n_neighbors=9"?

No, `n_neighbors` is unrelated to the number of topics. `n_neighbors` specifically controls how `UMAP` behaves when projecting the document vectors into lower dimensional space (see more [here](https://umap-learn.readthedocs.io/en/latest/parameters.html)). This value should be calibrated based on the number of documents you have in your sample. 

For discussion of clustering with `UMAP`, and how `n_neighbors` matters for clustering, see [here](https://umap-learn.readthedocs.io/en/latest/clustering.html).

**Comment by MrPaulAlbert** [comment]
Thanks!

**Comment by silkywork** [comment]
Hello, 

Although I read all the docs and the various issues related to my question, I can't figure out how to achieve what has been asked by @MrPaulAlbert. 

I created a Top2vec model that I saved. My understanding then is that I should load the model as such :

```
top2vec_model = Top2Vec(mymodel)
doc_vecs = top2vec_model.document_vectors

```
But I always get the error. : "ValueError: Documents need to be a list of strings". 

Sorry in advance as my question must sounds stupid but I am new to python and read everything possible trying to understand what is happening but couldn't figure out exactly what to do. 

Thanks for your help.
